app/code/Magento/Wishlist/view/frontend/templates/view.phtml

```diff
@@ -10,6 +10,9 @@
 ?>
 
 <?php if ($this->helper('Magento\Wishlist\Helper\Data')->isAllow()) : ?>
+    <?php if ($block->getPagerHtml()): ?>
```

Comment:
I think it will be better to render the block using `$block->getChildHtml('wishlist_item_pager')` - this approach will be consistent with the other code in the template and will eliminate the need to implement the `getPagerHtml` method

Reply:
done